lib/cli.js

```diff
@@ -257,6 +258,12 @@ WDIO Configuration Helper
         default: './',
         when: (answers) => answers.reporters === 'json'
     }, {
+        type: 'input',
+        name: 'outputDir',
+        message: 'In which directory should the mochawesome json reports get stored?',
+        default: './',
+        when: (answers) => answers.reporters === 'mochawesome'
```

Comment:
This is not quite right, I think `answers.reporters` is an array. Do you mind updating the two example above of this too?

Reply:
Feel free to write
```js
when: (answers) => answers.reporters.includes('mochawesome')
```

Reply:
Done